### PR TITLE
feat(tslint): add support to read extended `tslint.json`

### PR DESCRIPTION
**Pull request by alan-agius4**
Closes: #123 

I didn't add any new tests as the current tests already cover this part of logic

Maybe we shall add tslint as a peerdepence?

@vvakame as promised here's the PR :)

### CODE REVIEW

File: lib/provider/tslintjson.ts [hunk 1]
```diff
@@ -1,118 +1,97 @@
 import * as ts from "typescript";
-
+import { IOptions } from "tslint";
```
Comment (alan-agius4) [inline lib/provider/tslintjson.ts]: this is only used for interfaces thus not required as a dependency

**Comment by alan-agius4** [comment]
bump @vvakame

**Comment by vvakame** [comment]
@alan-agius4 sorry for late!
definitely nice!

**Comment by alan-agius4** [comment]
Thanks 😀

**Pull request merged**